.gitignore

```diff
@@ -0,0 +1 @@
+/mcuboot
```

Comment:
this seems a little weird.

Reply:
Is this correct?

Reply:
It was needed since west checked out to here. Removed now that mcuboot has moved to ../tfm-mcuboot